subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -78,6 +78,8 @@ The following breaking changes are not from deprecations, but the result of chan
 +
 While it applies the Java Plugin, it behaves slightly different (e.g. it adds the `api` configuration). Thus, make sure to check whether your build behaves as expected after upgrading.
  * The `html` property on `CheckstyleReport` and `FindBugsReport` now returns a https://docs.gradle.org/current/dsl/org.gradle.api.reporting.CustomizableHtmlReport.html[`CustomizableHtmlReport`] instance that is easier to configure from statically typed languages like Java and Kotlin.
+ * The `forkEvery` property of the `Test` task type now doesn't accept being set to `null`, use `0` to specify no maximum instead.
```

Comment:
:x: _The `forkEvery` property of the `Test` task no longer accepts `null`, ..._ ?